src/monitor/monitor_netlink.c

```diff
@@ -26,6 +26,9 @@
 #include <sys/types.h>
 #include <sys/ioctl.h>
 #define __USE_GNU /* needed for struct ucred */
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE /* needed by musl to enable struct ucred */
+#endif
```

Comment:
I've now pushed a commit that fixes that and also tried and verified that it all compiles like it should with that change.